itasks_server/websocket-sharp.xml

```diff
@@ -0,0 +1,7850 @@
+<?xml version="1.0"?>
```

Comment:
This file and the .exe do not belong inside this repository

Reply:
removed